src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonOutputFormatter.cs

```diff
@@ -25,16 +24,11 @@ public class JsonOutputFormatter : TextOutputFormatter
         // the serializer and invalidate it when the settings change.
         private JsonSerializer _serializer;
 
-        public JsonOutputFormatter()
-            : this(SerializerSettingsProvider.CreateSerializerSettings(), ArrayPool<char>.Shared)
-        {
-        }
-
-        public JsonOutputFormatter(JsonSerializerSettings serializerSettings)
-            : this(serializerSettings, ArrayPool<char>.Shared)
-        {
-        }
```

Comment:
That said, if others agree, I could restore the `JsonOutputFormatter(JsonSerializerSettings)` overload. Don't see any reason for the parameterless constructor.